docs/zh/navigation.yml

```
@@ -390,6 +390,8 @@ nav:
               - 背景: network/intro/background.md
               - 挑战: network/intro/challenges.md
               - 网络组件: network/modules/modules.md
+          - 规划:
+              - 网卡规划 : network/plans/ethplan.md
```

```suggestion
              - 网卡规划 : network/plans/ethplan.md
```
如果新建 chapter，至少要包含 2 节，孤岛生存不易

---

后续还需要补充其他内容